### PR TITLE
Chore - use xunit.abstractions instead

### PR DESCRIPTION
**Pull request by stijnmoreels**
Relates to #19

### CODE REVIEW

File: src/Arcus.Testing.Tests.Integration/Arcus.Testing.Tests.Integration.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```
Comment (tomkerkhove) [inline src/Arcus.Testing.Tests.Integration/Arcus.Testing.Tests.Integration.csproj:4]: Let's bump to 3.1 since 3.0 is already end of life

Comment (stijnmoreels) [reply src/Arcus.Testing.Tests.Integration/Arcus.Testing.Tests.Integration.csproj:4]: Ok!

**Comment by stijnmoreels** [comment]
Now a NuGet package is generated also, so we I can run again the release with the same `Package.Version` to publish that too. Or else, we can publish directly to NuGet but I don't have access to it direclty.

**Comment by tomkerkhove** [comment]
> Now a NuGet package is generated also, so we I can run again the release with the same `Package.Version` to publish that too. Or else, we can publish directly to NuGet but I don't have access to the NuGet account direclty.

@stijnmoreels  Feel free to just re-run with the same version!

**Pull request merged**